membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -43,8 +43,6 @@ object AccountDetails {
 
       val product = accountDetails.subscription.plan(catalog).product(catalog)
 
-      val mmaCategory = mmaCategoryFrom(product)
```

Comment:
this is unused and was removed in a previous PR from manage, so is also removed here now.